docs/http/openfga.http

```diff
@@ -0,0 +1,368 @@
+@fga_store_id = 
+@fga_model_id = 
+
+########################
+# STORES
+########################
+
+### List All Stores
+GET {{fga_api_url}}/stores
+Content-Type: application/json
+
+
+### Create Store
+# @name store
+POST {{fga_api_url}}/stores
+Content-Type: application/json
+
+{
+    "name": "openfga-demo"
+}
+
+### Save API token as a request variable
+@fga_store_id = {{store.response.body.id}}
+
+### Get A Store
+GET {{fga_api_url}}/stores/{{fga_store_id}}
+Content-Type: application/json
+
+
+### Delete Store
+DELETE {{fga_api_url}}/stores/{{fga_store_id}}
+Content-Type: application/json
+
+
+########################
+# AUTHORIZATION MODELS
+########################
+
+### Get All Authorization Models
+GET {{fga_api_url}}/stores/{{fga_store_id}}/authorization-models
+Content-Type: application/json
+
+### Get Authorization Model
+GET {{fga_api_url}}/stores/{{fga_store_id}}/authorization-models/{{fga_model_id}}
+Content-Type: application/json
+
+### Create New Authorization Model
+# @name model
+POST {{fga_api_url}}/stores/{{fga_store_id}}/authorization-models
+Content-Type: application/json
+
+{
+  "schema_version":"1.1",
+  "type_definitions": [
+    {
+      "type":"user"
+    },
+    {
+      "metadata": {
+        "relations": {
+          "owner": {
+            "directly_related_user_types": [
+              {
+                "type":"user"
+              }
+            ]
+          },
+          "parent": {
+            "directly_related_user_types": [
+              {
+                "type":"folder"
+              }
+            ]
+          }
+        }
+      },
+      "relations": {
+        "owner": {
+          "this": {}
+        },
+        "parent": {
+          "this": {}
+        }
+      },
+      "type":"folder"
+    },
+    {
+      "metadata": {
+        "relations": {
+          "can_view": {
+            "directly_related_user_types": [
+              {
+                "type":"user"
+              }
+            ]
+          },
+          "owner": {
+            "directly_related_user_types": [
+              {
+                "type":"user"
+              }
+            ]
+          },
+          "parent": {
+            "directly_related_user_types": [
+              {
+                "type":"folder"
+              }
+            ]
+          }
+        }
+      },
+      "relations": {
+        "can_view": {
+          "union": {
+            "child": [
+              {
+                "this": {}
+              },
+              {
+                "computedUserset": {
+                  "relation":"owner"
+                }
+              }
+            ]
+          }
+        },
+        "owner": {
+          "this": {}
+        },
+        "parent": {
+          "this": {}
+        }
+      },
+      "type":"document"
+    }
+  ]
+}
+
+### Save API token as a request variable
+@fga_model_id = {{model.response.body.authorization_model_id}}
+
+
+########################
+# /CHANGES
```

Comment:
Should the section corresponds to the section in https://openfga.dev/api/service?  (i.e., changes belong to "Relationship Tuples"?)

If what we have here makes more sense, should we update [our doc's swagger](https://github.com/openfga/openfga.dev/blob/754ab7dc2e9e774e63301f393cdc717bd67092a8/src/components/SwaggerUI/swagger-ui.tsx#L33-L37) to have the same breakdown?